src/registry.json

```diff
@@ -1960,9 +1963,9 @@
         "name": "Aleph Zero Wallet",
         "author": {
           "name": "Bide",
-          "website": "https://bide.dev"
+          "website": "https://bide.dev/"
         },
-        "website": "https://bide.dev",
+        "website": "https://azero.dev/",
```

Comment:
Is it intentional to bring users to a block explorer after installing the Snap?

Reply:
It was an Update request from the Builder to have this as the `website` for the Snap.

Reply:
It is, in fact, intentional